yb-voyager/cmd/exportData.go

```diff
@@ -109,6 +109,16 @@ func exportDataCommandPreRun(cmd *cobra.Command, args []string) {
 	if changeStreamingIsEnabled(exportType) {
 		useDebezium = true
 	}
+
+	if bool(source.AllowOracleClobDataExport) {
+		if source.DBType != ORACLE {
+			utils.ErrExit(color.RedString("allow-oracle-clob-data-export is only valid with source db type oracle. Remove this flag and retry."))
+		} else if changeStreamingIsEnabled(exportType) || useDebezium {
+			utils.ErrExit(color.RedString("allow-oracle-clob-data-export is not supported for live/BETA_FAST_DATA_EXPORT export paths. Remove this flag and retry."))
```

Comment:
Let's have a clear message of whether the user is in the Live migration / BETA_FAST_DATA_EXPORT, so have separate if conditions

Reply:
OR specify that it is only supported for normal offline migration

Reply:
@sanyamsinghal added separate conditions for live vs beta fast export. If you feel its better to just specify only supported for offline migration we can do that too?
I feel more explicitly telling the user that we dont support it in beta fast or live might be better for UX?